### PR TITLE
[ADD] delete_template_translations: New method for deleting translati…

### PR DESCRIPTION
**Pull request by rruebner**
…ons of changed templates with noupdate true.

@pedrobaeza Improvement of your suggestions in https://github.com/OCA/OpenUpgrade/pull/1195 and https://github.com/OCA/OpenUpgrade/pull/1188.

### CODE REVIEW

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -1668,3 +1669,26 @@ def rename_property(cr, model, old_name, new_name):
     cr.execute(
         "update ir_property set name=%s where fields_id in %s",
         (new_name, field_ids))
+
+def delete_template_translations(cr, module, model, xml_ids):
```
Comment (StefanRijnhart) [inline openupgradelib/openupgrade.py]: Prefer `delete_record_translations`

Comment (StefanRijnhart) [reply openupgradelib/openupgrade.py]: The model argument is void. You can take it from ir_model_data.

File: openupgradelib/openupgrade.py [hunk 2]
```diff
@@ -1668,3 +1669,26 @@ def rename_property(cr, model, old_name, new_name):
...
+    """Cleanup translations of updated templates with noupdate true.
+
+    :param module: module name
+    :param model: model name
+    :param xml_ids: a tuple of xml record IDs
+    """
+    if not isinstance(xml_ids, tuple):
```
Comment (StefanRijnhart) [inline openupgradelib/openupgrade.py]: Allow for lists as well, perhaps?

**Comment by pedrobaeza** [comment]
Any news with this?

File: openupgradelib/openupgrade.py (hunk 1 above)
Comment (rruebner) [reply openupgradelib/openupgrade.py]: Done.

File: openupgradelib/openupgrade.py (hunk 2 above)
Comment (rruebner) [reply openupgradelib/openupgrade.py]: Done.

**Pull request merged**